go/vt/topo/consultopo/server.go

```diff
@@ -36,6 +36,7 @@ import (
 
 var (
 	consulAuthClientStaticFile = flag.String("consul_auth_static_file", "", "JSON File to read the topos/tokens from.")
+	maxConnsPerHost            = flag.Int("consultopo_max_conns_per_host", 0, "limit the MaxConnsPerHost used in the consul client's http Transport. Zero means no limit.")
```

Comment:
might be worth adding a comment around why this exists. If i were to see this without our context, I wouldn't be sure if i need to set it or not.


general Q: do we have any idea how big this gets in practice(i.e. what number of conns tips us over the edge)?